blueprints/ember-decorators/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  normalizeEntityName: function() {}, // no-op since we're just adding dependencies
+
+  afterInstall: function() {
+    return this.addPackagesToProject([
+      { name: 'babel-plugin-transform-class-properties' },
```

Comment:
I think it's ok to include both, the current examples we're laying out require both and as the plugins are updated we can transparently keep it working without users needing to know the details. I think maybe adding a flag to allow users to explicitly opt out may be a better solution, that way we won't throw warning if they're using a different version of babel either.